boards/arduino-uno/Makefile.include

```diff
@@ -0,0 +1,46 @@
+# define the cpu used by the arduino uno board
+export CPU = atmega328p
+
+export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
+export TERMFLAGS = -b 9600 -p $(PORT)
+
+#define the flash-tool and default port depending on the host operating system
+OS = $(shell uname)
+ifeq ($(OS),Linux)
+  PORT ?= /dev/ttyUSB0
```

Comment:
It is necessary to separate CPU concerns from board concerns. The atmega-common is intended for CPU. Here the need is a common platform source, from where arduino-based boards can leverage the common code